modules/generator/builder.tmpl

```diff
@@ -0,0 +1,21 @@
+package modules
+
+import (
+    {{- range $pkg := .}}
+    "github.com/prebid/prebid-server/modules/{{$pkg}}"
+    {{- end}}
+)
+
+func init() {
+    builderFn = builders
+}
```

Comment:
This was necessary if we didn't include the `modules/builder.go` file in source control. But I'll change that since we've decided to keep it inside source control.